src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -396,12 +403,8 @@
         It is recommended that you supply your parameters file first, and then override selectively using KEY=VALUE syntax.
 examples:
   - name: Validate whether a template is syntactically correct. (autogenerated)
-    text: |-
-        az group deployment validate --parameters '{
-                "location": {
-                    "value": "westus"
-                }
-            }' --resource-group MyResourceGroup --template-file storage.json
+    text: |
+        az group deployment validate --parameters '{         "location": {             "value": "westus"         }     }' --resource-group MyResourceGroup --template-file storage.json
```

Comment:
The original format of Json has been flattened out,  which is not good for the user to see the parameters and data structure. Can we keep the original format here?

Reply:
Good spot!
I will find a way to take care of this multi lines examples.